app.js

```diff
@@ -0,0 +1,104 @@
+var Issue = can.DefineMap.extend({
+  seal: false
+}, {
+  id: 'number',
+  title: 'string',
+  sort_position: 'number',
+  body: 'string'
+});
+
+Issue.List = can.DefineList.extend({
+  '*': Issue
+});
+
+Issue.algebra = new can.set.Algebra(
+  can.set.props.sort('sort')
+);
+
+Issue.connection = can.connect.superMap({
+  url: '/api/github/repos/chasenlehara/webhook-test/issues',// TODO: this will be updated to canjs/github-issues-example
+  Map: Issue,
+  List: Issue.List,
+  name: 'issue',
+  algebra: Issue.algebra
+});
+
+var socket = io();
```

Comment:
It doesn’t because socket.io is loaded from the server, so it knows how to connect.